Stripe/Resources/Localizations/ko.lproj/Localizable.strings

```diff
@@ -0,0 +1,267 @@
+/* {card brand} ending in {last4} */
+"%@ Ending In %@" = "%2$@(으)로 끝나는 %1$@ ";
+
+/* Source type brand name */
+"3D Secure" = "3D Secure";
+
+/* part of {Account} ending in {last4} */
+"Account" = "계좌";
+
+/* Title for address entry section */
+"ACCOUNT HOLDER ADDRESS" = "계좌 소유자 주소";
+
+/* Caption for Name field on bank info form */
+"Account Holder Name" = "계좌 소유자 주소";
+
+/* Title for Add a Card view */
+"Add a Card" = "카드 추가";
+
+/* Title for SEPA Debit Account form */
+"Add a Direct Debit Account" = "직불 카드 계좌 추가";
```

Comment:
Translating this as "Debit Card" seems off to me, as it's more of a bank transfer? Same comment for all the SEPA strings.

Reply:
was about to reach out to localization but...turns out we don't even use this anywhere (or else it would have shown up in a screenshot sent to them). Gonna just not block for now and we can do a pass on unused keys